### DISCUSSION TITLE
日本語もイタリック体に出来るようにして欲しい

### DISCUSSION THREAD

**Discussion by lamrongol** [increments/qiita-discussions#267] [要望・アイデア・不具合]
[Markdown記法 チートシート \- Qiita](https://qiita.com/Qiita/items/c686397e4a0f4f11683d)には「_ か * で囲むとHTMLのemタグになります。Qiitaでは イタリック体 になります」とありますが、半角英数はイタリック体になりますが日本語はイタリック体になりません（例：[オブジェクト指向とは結局何なのか あるいはプログラミングで気をつけるべきたった一つのこと \- Qiita](https://qiita.com/lamrongol/items/8a339721ee20a1c81e3c)の「（即物的に言えば）」の部分）。
なおFirefoxではイタリック体になるのですが、Chromeではイタリック体になりません。

**Comment by shiracamus** [comment]
HTML5 で `<em>` タグの仕様が変わったのが影響しているのかな？
`<i>` タグにするといいのかもしれませんね。

**Comment by tanaka-asami415** [comment]
@lamrongol 
お問い合わせありがとうございます🙇
OSを教えていただきたいのと、
フォントによっては、日本語のイタリック体がない場合もあるため、お使いのフォントを知りたいです。
お手数ですがスクリーンショットをいただけないでしょうか🙏

@shiracamus 
ありがとうございます🙏✨

**Reply by lamrongol** [reply]
OSは Windows 10 Home 64bit
フォントはメイリオのようです。
![image](https://user-images.githubusercontent.com/4645074/182279907-a8540278-2ec8-4526-9f6d-95f9281a4b18.png)
![image](https://user-images.githubusercontent.com/4645074/182278411-5576d29b-0655-48b4-a3b7-b80cc9acb923.png)
ちなみにどうもChromeはuser agent stylesheetでemを打ち消しているようです。
![image](https://user-images.githubusercontent.com/4645074/182280091-0ec1153e-6f0c-4688-9740-3e71838a9165.png)

**Reply by tanaka-asami415** [reply]
@lamrongol 
情報のご提供ありがとうございます🙏

lamrongolさんの環境では、Qiitaサイトがメイリオで表示されているようですが、
メイリオは斜体データを持っていないため、斜体表示されないことが原因のようです。

emは打ち消されていないのですが、上記の理由によりイタリック表示になりません。
Firefoxで斜体になるのは、ブラウザの仕様により、
斜体データを持っているMSゴシックに置き換えられて表示されるためだと思われます。

ご不便とご迷惑をおかけして申し訳ありません。
今後改善できるよう検討いたしますが、一旦「Markdown記法 チートシート - Qiita」の記載内容を修正いたします🙇

**Reply by lamrongol** [reply]
stackoverflowで日本語のみイタリック体にできないか（正確には日本語しか定義されてないフォントはないか）質問したところ以下のように二つ回答と一つのページが寄せられたのですが、これで対応できるでしょうか？
https://ja.stackoverflow.com/questions/90391/%E8%8B%B1%E6%95%B0%E3%81%AF%E5%AE%9A%E7%BE%A9%E3%81%95%E3%82%8C%E3%81%A6%E3%81%8A%E3%82%89%E3%81%9A%E6%97%A5%E6%9C%AC%E8%AA%9E%E3%81%97%E3%81%8B%E5%AD%98%E5%9C%A8%E3%81%97%E3%81%AA%E3%81%84%E3%83%95%E3%82%A9%E3%83%B3%E3%83%88%E3%81%AF%E3%81%82%E3%82%8A%E3%81%BE%E3%81%99%E3%81%8B
https://www.softel.co.jp/blogs/tech/archives/5673

**Reply by tanaka-asami415** [reply]
@lamrongol 
ありがとうございます🙏
参考にさせていただきます🙇

**Reply by Phroneris** [reply]
lamrongolさんと同様、Qiitaで日本語が斜体表示されず不便しています。
Windows 10、Chromeです。メイリオで表示されています。
また、Firefoxだと確かに斜体表示してくれます。

メイリオには斜体が無いというより、**斜体の領域に正体（せいたい）の字形がわざわざ登録されている**と聞き及びます。
また、lamrongolさんの質問に寄せられた `transform` プロパティを使う方法は、そのページにも書かれている通り複数行に対応できないのできっとダメでしょう。

WindowsユーザーかつChromeユーザーとしては、**斜体の利用を想定したサイトでメイリオなんて使わないでくれ～！！！** ***本当に～！！！！！*** と叫びたいのですが…。

ただ、少しググりかじった限りですがこういう解決法もあるようです。→ https://qiita.com/IrakaT/items/7e5623357ba2b8b560de
実際、Chrome拡張機能[Stylus](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne)で次のようなユーザーCSSをQiitaに適用してみたところ、確かにメイリオのまま斜体表示されました。

```css
@font-face {
  font-family: "Meiryo Italic";
  src: local('Meiryo'), local('メイリオ');
}
.it-MdContent em {
  font: italic 100% "Meiryo Italic", sans-serif;
}
```

※ 元記事のままフォントファミリー名を `Meiryo` とすると日本語の太字字形にまで影響が出たので、当該部分は `"Meiryo Italic"` に変更しました。

参考になれば幸いです！

**Reply by Phroneris** [reply]
その後、首尾はどうですか？

Markdown記法チートシートがどう修正されているのかと思えば、斜体になるべき部分を "italic type" と英字に置き換えているだけで、メイリオに対しての注意喚起は皆無…。
少しガッカリです。

厳しい事を言えば、斜体のあるサイトでメイリオを強制しているQiita側の落ち度でしかないこの問題に未だ何の解決も見せず、Markdown記法のヘルプにあたる記事までもがこのように誤魔化すだけで既知の問題があることすら明示せず、対応策といえばメイリオの斜体問題というピンポイントなリテラシーをユーザー個々人にどこかから自力で仕入れて貰うぐらいしか無い事にただ任せて何年も放置しているというこの状況は、ユーザーとして快く受け入れられる物ではありません。
QiitaはMarkdown変換サービスではなく、Markdownを手段として記事を執筆し共有するサービスのはずです。
Markdownの最も基本的な機能の1つとして知られる `*斜体*` を、仕様通り `<em>斜体</em>` にレンダリングしてハイ終わり！ ほんとに斜めになるかは知らん！ ヒラギノを持ってない奴が悪い！ …これって記事執筆サービスのあるべき姿ですか？
いやまぁ、もちろん諸悪の根源はメイリオ自体とそれを普及させたWindowsとかにはなりますけど、でもそれを知りながら今も無策で採用し続けているのはQiita自身ですよね。

とりあえず、私の叫びを再掲します。
**斜体の利用を想定したサイトでメイリオなんて使わないでくれ～！！！** ***本当に～！！！！！***

**Comment by tats-u** [comment]
個人的にはメイリオ強制はやめて`sans-serif`でユーザ任せにしてもらいたいです。
メイリオよりNoto Sans/源ノ角の方が絶対見やすいです

最近はこのような改善もありましたし。

https://developer.chrome.com/blog/better-text-rendering-in-chromium-based-browsers-on-windows?hl=ja

`<em>`に関してはユーザごとのオプトインで圏点に変更できると理想ですが。

```css
em {
  font-style: normal;
  text-emphasis: circle; /* またはdot */
}

**Comment by P-SiZK** [comment]
@Phroneris @tats-u 
ご意見ありがとうございます！また、長らくお待たせしてしまい大変申し訳ありません :bow:

以前お知らせいただいたフォントの問題についてですが、メイリオを含め指定を見直す対応を進めていきたいと思います。

圏点に関するご提案もありがとうございます！
今後の改善の参考にさせていただきます。

ご不便、ご迷惑をおかけして大変申し訳ありませんが、対応まで今しばらくお待ちいただけますと幸いです :pray:

**Reply by tats-u** [reply]
ご検討ありがとうございます。emを圏点に変えるだけの他にも、emで普通の太字、strongで色文字（典型的には赤、下品なら他の色）や蛍光マーカー風やただの色線を太字に加えて付与、両方指定してstrong単体と変わらないなら圏点やマーカー追加で付与　という案もあります。いくつか選択できるようにできるのが理想ではあります。

**Reply by Phroneris** [reply]
前向きにご対応頂けそうでありがとうございます。宜しくお願い致します。
まぁ、何事も全てに配慮した完璧な対応は難しい物だろうとは思います。アクセシビリティとか色覚多様性とかありますし。
ただ何にせよ、現行ではWindowsとChromeを使っているだけで基本機能すらオミットされてしまっている状態ですので、改善を強く要求させて頂きました。

少なくともMarkdownチートシートの斜体記法の箇所には明示すべきではないでしょうか？
記事編集よりも指定変更の方が早いなんて事は無いでしょう？

**Reply by P-SiZK** [reply]
@Phroneris 
ご返信ありがとうございます！

いただいたご提案に基づき、まずは「Markdown記法 チートシート」に注意書きを追記いたしました。

https://qiita.com/Qiita/items/c686397e4a0f4f11683d#emphasis--strong-emphasis---%E5%BC%B7%E8%AA%BF%E5%BC%B7%E5%8B%A2

根本的なフォント指定の見直しにつきましても、引き続き対応を進めてまいります。

**Reply by tats-u** [reply]
ちなみに`_`は日本語では基本的反応しないので、例示するなら`*`1個の方を優先すべきです。日本語での`_`は段落全体を囲む用途専用です。

前後に半角スペースまたは改行を入れてください→前後に半角スペースまたは改行を入れない場合、反応しない場合があります or 反応しない場合は、前後に半角スペースまたは改行を入れてください

**Comment by TETSURO1999** [comment]
### 今すぐ圏点を表現したい人へ

`<ruby>` タグを使えば可能です。コチラは **github** でも使えますね。

おそらく<ruby>圏点<rt>●●</rt></ruby>ってこんな感じですか？

```html
おそらく<ruby>圏点<rt>●●</rt></ruby>ってこんな感じですか？
```

### 今すぐ日本語で斜体表現したい人へ

**MathJax** を使えば可能です。残念ながら **github** では対応していない様子。

```html
日本語で$\textit{斜体}$ってこんな感じですか？
```

インラインではなくブロックで良いのなら **PlantUML** を使えば可能です。詳しくは下記の記事を参照下さい。

https://qiita.com/TETSURO1999/items/bd6252bab2f24bb38d0e

### 日本語がイタリック体にならない理由を整理する

まず，**Qiita** は記事の主要フォントとしてメイリオを指定しています。

**Windows** ではメイリオにイタリック体とボールド体のスタイルを用意していますが，標準スタイルと全く同じ字形なのが諸悪の根源です。もしも，これらのスタイルが存在しなければ **Windows** が標準スタイルをベースにして斜体や太字を作ってくれるからです。

**Mozilla Firefox** は既定フォントとしてメイリオを指定しているせいか，アプリケーションレベルでの工夫によりメイリオでの斜体を実現しています。しかし，最大派閥の **Chrome** 系は既定フォントを自社の **Noto** 系に切り替えているせいか，このような工夫は特に行っていません。このためメイリオのイタリック体を指定しても標準スタイルと字形が変わりません。

### Qiita 運営への提言（遅きに失した感はありますが）

**Qiita** の記事内で使われているフォントですが，**CSS** 設定を見てみると `<body>` タグの `font-family` は下記の順で設定されています。

|フォント|OS|
|:--|:--|
|**YakuHanJPs**|**Web**フォント|
|**-apple-system**|**macOS**|
|**BlinkMacSystemFont**|**macOS**|
|**Segoe UI**|**Windows**|
|**Hiragino Kaku Gothic ProN**|**macOS**|
|**Hiragino Sans**|**macOS**|
|**Meiryo**|**Windows**|
|**sans-serif**|上記以外のとき|

このうち **Windows** では **Segoe UI** と **Meiryo** が使われることになりますが，いずれも幻の **Windows Vista** で採用されたフォントです。**Windows 11** 時代の現在においては **Noto Sans JP** 等のより優れたフォントが開発されており，それらを直接指定しても良いと思いますが，コードブロックのほうは単に **monospace** とのみ指定されており，具体的なフォント選択はブラウザ任せとなっていることから，本文のフォントも同様に **sans-serif** として具体的なフォント選択はブラウザに委ねても良いように思います。

ちなみに編集画面の行番号では **Consolas** を指定しており，0とOの区別がし易く個人的には好みなのですが，日本語とのマッチングが難しいのでコードブロックに使わなかったのではないかと推察しています。


以下，**Windows** における主要ブラウザの既定フォントを示します。

||**Chrome**|**Firefox**|
|:--|:--|:--|
|明朝体|**Noto Serif JP**|遊明朝|
|ゴシック体|**Noto Sans JP**|メイリオ|
|等幅|**BIZ UD**ゴシック|**BIZ UD**ゴシック|

既定フォントにすれば **Chrome** 系は問題なく日本語でも斜体を使えるようになり，もともと **Firefox** はメイリオでも斜体を実現できていたので問題ありません。

**Reply by tats-u** [reply]
rubyでやるのは読み方を●●で上書きしているようなものです。日本語でイタリック使って強調はナンセンスだからもっと適切な方法で置き換えましょうよ、というだけです。

https://developer.mozilla.org/ja/docs/Web/HTML/Reference/Elements/em

**Reply by Phroneris** [reply]
個人的には斜体での強調がそんなにナンセンスだとは思いませんね…。
斜体なんてMicrosoft Office日本語版にも昔からあって実際日本語圏で使われてきてるんですし（英語圏程じゃないでしょうが）、少なくとも普通の文字よりは何か特別な意図を持って変形させていると考えるのが自然ですから、使われて意味が全くわからず困惑するような人はそんなに多くないんじゃないかなと思います。

圏点での強調については、等幅ベースの日本語文字だけなら等間隔で配置されるので確かに馴染みやすいですが、それでもフォントによっては「っ」[^1]とか半角カタカナとか必ずしも等幅じゃない文字もありますし、ましてや英数字なんかプロポーショナル大前提なのでさすがに不適じゃないでしょうか。
CSSの圏点機能は、あくまで文書のコンテンツと表示スタイルを全て掌握できる自サイトの作り手のために用意されている物という感じがします。
Qiitaのようなユーザーにコンテンツしか制御させられないサービス、しかもIT系である以上英数字との混ぜ書きを強調する場面なんて絶対多くあるサービスにおいては、適切な圏点装飾というのは余程慎重にやらない限り難しいかなと。

それよりはマーカー下線の方が文字種を選ばず直感的にできそうですね。
でもやっぱり、私は「別に斜体でええやん」派でございます

[^1]: メイリオでは「っ」も等幅ですけどね

**Comment by katouriko** [comment]
@lamrongol @shiracamus @Phroneris @tats-u @TETSURO1999
大変お待たせしました :bow: 
Windows版Google Chrome の環境でイタリック体が反映されるよう、Meiryo よりも以下のフォントを優先して使用するように設定変更いたしました。

- Noto Sans JP
- BIZ UDGothic

Meiryoの指定は、Noto Sans JP,BIZ UDGothic が当たらなかった場合も考慮して残しております :pray:
なお、フォント設定は、今後もトレンドや皆様のフィードバックを反映し、継続的に改善していきます。 引き続きご意見いただけますと幸いです :memo:

[リリースノートはこちら](https://qiita.com/release-notes/2025-11-06)

今後とも Qiita をよろしくお願いいたします :pray:

**Reply by Phroneris** [reply]
@katouriko
ご英断ありがとうございます！！ *文字傾け放題！！！*<sub>（GitHubでやっても意味無い）</sub>
きっとこの変更に対し色々と新たな意見が噴出するとは思いますが、そのような方々にも是非このディスカッションを参照するようご案内頂ければと思います。
みんな語ろうぜ！ 親指だけでも上げ下げして行ってね！ ヘイヘイ！

で、早速なんですが私からも1つ意見を…。

まず現状を確認すると、現在のフォント指定は `font-family: YakuHanJPs, -apple-system, BlinkMacSystemFont, "Segoe UI", "Hiragino Kaku Gothic ProN", "Hiragino Sans", "Noto Sans JP", "BIZ UDGothic", Meiryo, sans-serif;` となっているようですね。
つまり多くのWindows環境において日本語文字には【YakuHanJPs > Noto Sans JP > BIZ UDゴシック > メイリオ】という優先順位が適用され、Yaku Han JPは別としてNoto Sans JPとBIZ UDゴシックはいつからかWindows 10/11に標準搭載されるようになったっぽいので、恐らくほとんどのWindowsユーザーはNoto Sans JPで日本語文字が表示されるのを目にすることになるのでしょう。

でも、**Notoフォントの存在意義**ってあくまで**多言語想定の文書におけるスタイルの統一と「豆腐」の回避のためのフォールバック**じゃないでしょうか？
Qiitaにメイリオが永らく採用されて来たのはメイリオの見やすさ・美しさが評価されていたためだと思うのですが（完全に想像ですけど）、Noto Sans JPは多言語の統一と網羅を主目的とする誕生の経緯のせいか、日本語専用に作られたメイリオやBIZ UDゴシックに比べてそのような観点ではどうしても一歩及ばないというのが少なくない意見だと思います。
ぶっちゃけ、Noto Sans JPの見やすさ・美しさは**いわゆる「中華フォント」とメイリオやBIZ UDゴシックとの間ぐらい**、前者よりはマシなだけで後者には及ばない、というのが私個人の感覚です。
というか今はそれこそ中華フォントが表示されるような状況でフォールバックされるのがNotoなんでしょうけど。

そういう訳で、日本企業がほぼ日本語話者のためだけに運営しているQiitaにおいては、日本語文字のフォントの優先順位は**BIZ UDゴシックをNoto Sans JPより優先する**ように変更する方が、前のメイリオの見やすさからより自然に受け入れられる変更になると思います。
現状の一利用者としての印象は、わざわざ見辛い方のフォントに変わってギョッとさせられるという感じで、正直あまり良くないです。
敢えて反感を買うように変更した訳でもないでしょうから…。

まぁ、もっと言うと「っ」とかがプロポーショナルになるBIZ UD**P**ゴシックが指定されればというのが個人的に最大の欲を出した意見ですが、前のメイリオは「っ」とかも等幅だったので、日本語文字は基本的に等幅で行くというのがQiitaの流儀なんだなという事で私はそこは普通に受け入れられます。
あれを叶えてやったら次はこれも叶えろと永遠に踏み込んで来る厄介利用者と思われそうで大変恐縮ですが、本件については本当にあとここだけが何とかならないかと思う所なんです。
何卒ご検討をお願いします。
そしてこれを見た他の方々、これについてのご意見や親指をお待ちしています！

**Reply by tats-u** [reply]
外国語版Windowsだと游ゴシック系しか入っていませんが、扱いが難しいので無理に入れなくても構いません。
メイリオとNotoとBIZ UDは日本語補助フォントというセットに同梱されています。日本語版Windowには入っていますが、外国語版だと別途入れなければなりません。
メイリオはちょっと細いですし何かギザギザしてて字形も読み手を舐め腐っているような印象を受けるので下の方に行ってよかったです。
Notoは正統派のデザイン、かつプロポーショナルメトリクス対応でBIZ UDPみたいに字詰めできる、かつ太字のインパクトが大きい(かなり大事)ので最優先でよいです。
~~Qiitaだと他のサイトより妙に細い？~~（[ダークモードに限り細くしているようです](https://github.com/increments/qiita-discussions/discussions/267#discussioncomment-14981090)） とはいえフォントやその順番にあれこれこだわるよりはsans-serif一本で利用者が一番読みやすいと思うフォントにするのが一番いいと思いますが。

**Reply by katouriko** [reply]
@Phroneris @tats-u
ご意見ありがとうございます :bow:

「Noto Sans JP」と「BIZ UDGothic」のどちらを優先するかについては、デザインの好みや視認性の感じ方など、個々人で意見が分かれる部分かと思います :pray: 

そのため、一旦現状のまま様子を見させていただき、もし今後「Noto Sans が見づらい（視認性に課題がある）」といったご意見が多く寄せられるようでしたら、その時点で改めて検討いたします。

**Reply by tats-u** [reply]
BIZ UD（やメイリオ）の弱点として、Firefox以外で見たときに「（）」・「【】」などの並びの無駄な空白を削る処理が働かない点があります。（YakuHanJPsの需要を下げられます）
これが効くWindows内蔵ゴシックフォントはNoto Sans JPと游ゴシックしかないと思います。

**Reply by nogajun** [reply]
#1035 の投稿から誘導されたので来ました。  

Linuxユーザー視点でいうと、BIZ UDフォントよりNoto CJKフォントのほうが読みやすいです。
そして、WindowsでNoto CJKフォントの視認性の問題は、**Windowsのフォントレンダラーが貧弱**なことが原因でフォントの問題ではありません。ですので、自分の意見としては見づらいBIZ UDフォントより、Linuxで美しくレンダリングされるNoto CJKフォントを優先させてほしいです。

と、自分の意見を述べましたが、争いは避けたいので折衷案を考えてみました。この並びでどうでしょうか?

```
Noto Sans CJK JP, BIZ UDPGothic, Noto Sans JP
```

Noto CJKフォントはフォント形式によって微妙に名前が違います。これを利用してDebian/Ubuntuその他Linuxで使われている`Noto Sans CJK JP`を先に持ってきます。Linuxではこれが最初にマッチします。Windowsでは自分でインストールしない限りこのフォントは存在しないので、マッチせずフォールバックします。

Windowsでは、フォールバックするので`BIZ UDPGothic`が最初にマッチします。そして`Noto Sans JP`がマッチするという流れになるのでお互いハッピーになるのでは思いました。あとは、Windows側だけの問題ですね

（個人的には、OpenTypeオプションできれいな文字組みができるNoto CJKフォントを優先するといいとは思いますが、好みの問題になると思います。BIZ UDフォントは形式こそOpenTypeフォントでも実質TrueTypeフォントですからオプションが使えないんですよね…）

**Reply by Phroneris** [reply]
> ```
> Noto Sans CJK JP, BIZ UDPGothic, Noto Sans JP
> ```

これ面白いですね～！
私はLinuxの世界には詳しくないのですが（なのでLinuxならNotoが美しいと聞いて衝撃です）、お話が事実ならこれに1票投じたいです！

**Comment by honey32** [comment]
便乗ですみません。

Noto Sans JP の優先度が上がった後の 2025/11/16 現在のところ、ダークモード時に限り本文の `font-weight` が `300` になっていますが、

個人的に細すぎて不自然に感じます。何か意図があってのことかもしれませんが、検討のほどお願い致します 🙇 

- Windows 11 
- Google Chrome 142.0.7444.163
- 描画されているフォント（和文向け）Noto Sans JP

あまり参考にならないと思いますが、スクショを載せておきます。

| Before | After <br/> `normal` に上書き|
| -- | -- |
| <img width="854" height="242" alt="image" src="https://github.com/user-attachments/assets/e0086846-361c-467d-9878-fd8cfd9a910d" /> | <img width="855" height="245" alt="image" src="https://github.com/user-attachments/assets/75fa17d8-ba0a-47e0-a742-d93850719c52" />  |

**Reply by shiracamus** [reply]
私も、文字がかすれて読み辛くなったなぁと感じておりました。
私の環境ではダークモードだけじゃなくライトモードでも細いです。

**Reply by kimkim0814** [reply]
@honey32
コメントありがとうございます！

`font-weight: 300` の挙動ですが、意図的な仕様となっております。 
ダークモードの場合、通常の太さ（400）と太字（700）の差が視覚的にわかりにくくなってしまうことがあり、あえて `300` を指定することでメリハリをつけていました。

ただ、今回いただいたように「細すぎて不自然」と感じるのとても理解できます。
運営としても現状が最適解だとは言い切れていないため、見にくいユーザーが多い場合は太さを再度調整するなどして対応も考えています。

もし良いアイデアがあれば、ご意見いただけますと幸いです。

**Reply by honey32** [reply]
基本的には、優先順位の問題なので、`font-weight: 300` 設定を削除しても問題ないと思います。

- 通常のテキストが全て見栄えが悪いのは、他のサイトと比べて明らかに体験が悪い
- 「通常のテキスト / strong のテキストの差がわかりにくい」問題があるとしても、特に他のサイトにリードを許すことにはならないはず。
    - 日本語サイト全般が Windows 環境（特に非 Retina 環境）で抱えている問題なので。
    - Zenn でも普通に区別がつかない
        - これは、Mac の Chrome なら少し改善します
    - React 公式ドキュメント日本語版も同様です。https://ja.react.dev/learn/you-might-not-need-an-effect
        - こちらは、Mac / Chrome から見ても改善しません

一つの解決方法の案として、不透明度を変えて強弱をつける（ついでに、見出しも strong と同じ「強い」色にする）、という方法はありかもしれません。

| font-weight をリセットする | font-weight をリセット<br/>+ strong の不透明度: 1 |
| -- | -- |
| <img width="776" height="296" alt="" src="https://github.com/user-attachments/assets/3c38de9d-35ba-40c2-b589-0fbe6ef35dde" /> | <img width="770" height="296" alt="" src="https://github.com/user-attachments/assets/5ee6dd3f-d967-48a0-b613-98ccd3f959ae" /> |


---

参考までに、僕の個人サイトでは、思い切って strong と見出しの文字を強調色（ダークモードなら金色、ライトモードなら緑色）に設定しています。

UGC であり、かつリンク文字色がブランドカラーで上書きされている Qiita には適用できないと思うので、不透明度で差をつけるくらいでちょうど良い塩梅なのではないでしょうか？

https://www.bearwapps.com/posts/keigo-goissho-shimasu

| ダーク | ライト |
| -- | -- |
| <img width="815" height="353" alt="参考画像。ダークモードの画面で、基本の文字は黒、strong は緑色" src="https://github.com/user-attachments/assets/2028c4d5-d4fc-48c2-85b6-22546a52d9ef" /> |  <img width="846" height="306" alt="参考画像。ライトモードの画面で、基本の文字は白、strong は金色" src="https://github.com/user-attachments/assets/a656c666-051b-49b2-89cd-1b38bae74151" /> |

**Reply by tats-u** [reply]
そもそも、Noto自体が太字と通常のメリハリがついている方です。
BIZ UDや游だともっと差が小さいと思います。
普段見慣れないフォント/ウェイトが当ててあるともしかして中華フォントなんじゃないかと疑ってしまいます。

**Reply by Phroneris** [reply]
通常の文字に対して `*` や `**` による強調の区別が明確な事と、通常の文字が読みやすい事、どちらかしか取れないとしたら後者の方が大事だと思います。
だって通常の文字の方が多いんですから。
（この理屈だと「じゃあ最悪 `*` で斜体にならなくても良いじゃん」となっちゃうんですが、いやさすがに何も区別がつかないのは後者におけるマジモンの最悪なんで…）

まぁ色んな手段を考慮すればそもそも二者択一ではないのが当然ですけど、しかしあまりクセのある強調をするとMarkdownの素直な変換を期待する人間の感情面に差し障るのが本当に難しいですね…。
CSSの仕様的には「どうしても思い通りにしたきゃWebフォントでやれよ」なんですかね。文字の少ねえ民族中心の仕様がよォ～！！（感情）

**Reply by tats-u** [reply]
> 不透明度で差をつけるくらいでちょうど良い塩梅なのではないでしょうか？

普段の文字色は真っ白（#fff）・真っ黒（#000）ではないので`<strong>`の際にそれらに近づける余地が少しだけですがありますね。

> 後者の方が大事だと思います。

まさしくその通りです。どうしてもというのなら見出しと太字以外のフォントとしてNotoより細い游ゴシック Mediumが候補に上がります

**Reply by kimkim0814** [reply]
@honey32 @tats-u @Phroneris
コメントありがとうございます🙌

いただいたご意見を踏まえて社内で再検討し、今回は `font-weight: 300` の指定を削除する（標準に戻す）方針 で進めることにしました！

まずは「通常のテキストの読みやすさ」を優先に対応したいと思います。
不透明度での調整やフォント選定などのいただいたアイデアも、今後の改善の選択肢として大切に参考にさせていただきます☺️

対応まで少々お時間をいただくかもしれませんが、リリースをお待ちください！

**Reply by moroi** [reply]
@honey32 @tats-u @Phroneris
ご意見をいただいておりました `font-weight: 300` の指定につきまして、削除の対応が完了いたしました！
その他の点につきましても引き続きブラッシュアップを続けてまいります。
よろしくお願いいたします！ :bow:

**Comment by Phroneris** [comment]
ちなみに似て非なるディスカッションで、`等幅フォント` を和英比 2:1 が保証されるプログラミングフォントに変更した結果、不評の方が多くて元に戻しちゃった、なんて事もありましたね。→ https://github.com/increments/qiita-discussions/discussions/302#discussioncomment-4793132
メインフォントすら変えられるならあっちも再び検討されると嬉しい所ですが…（強欲）、でもそれ言うと逆にメインフォントまで差し戻されかねないな…（逃げ腰）

**Reply by gilly135** [reply]
@Phroneris 
ご意見ありがとうございます。
その節はご希望に添えず申し訳ありませんでした。

フォントは、皆様の環境や好み、またライセンスに左右されるところが多く難しいと感じております。

まずは今の時代に、できるだけ使いやすいと感じていただけるような日本語フォントを模索したいと思っております。

まとめて見直しできたら…という気持ちはあるのですが、一つずつ丁寧に応えていきたいと思っております 🙇

**Comment by tsubolabo** [comment]
最近、「見出し」の様式が変わったように感じるのですが、これもフォント変更の影響ですか？

具体的には：

今までは、下記の方法で見出しを書いており、これで程よいバランスがとれていました。

+ 全角の数字で章番号
+ 全角のドット
+ 見出しの一文字目

しかし、最近は章番号と見出しの一文字目の間隔が狭くなったように思えます。見出しの一文字目が数字になる場合には特に気になります。タイトルと章番号が混然一体になって見分けがつかなくなります。（本文として示した文字間には思い通りのスペースがとれています）

![](https://github.com/user-attachments/assets/810cf13f-d923-4cdb-ac57-fc2c500fa30e)

**Reply by Phroneris** [reply]
私もこれ地味に気になってました。
見出しとそれ以外とで、約物のスペース感が見るからに違います。

日本語の字詰めをプロポーショナルメトリクスに依存するNoto Sans JPがメインフォントで、更に何らかの理由により見出しにだけ `font-feature-settings: "palt";` がオプトインされているのが原因な訳ですが、現状ではこの細工で**「わぁ、見出しが見やすい！」**となるよりは**「うわ、統一されてない…」**となるのが個人的な本音です。
これ本文でもやったら駄目なんですかね？

<img width="900" height="600" alt="font-heading-space" src="https://github.com/user-attachments/assets/ebb6be91-24a6-4514-a1a0-86f18608987f" />

```markdown
# h1：あ【い】う
## h2：あ【い】う
### h3：あ【い】う
#### h4：あ【い】う
##### h5：あ【い】う
###### h6：あ【い】う

本文：あ【い】う
```

**Reply by tats-u** [reply]
中途半端な状態なので記事タイトル名もpaltまたはkernでツメ組にすべきです
個人的には見出しのツメ組には賛成です

**本文は今まで通りベタ組でいいです**、system-ui指定によるYu Gothic UIで感覚が麻痺している人も多いですが、長文ではベタ組の方が読みやすいとされています

https://note.morisawa.co.jp/n/n9b07908be089

paltまたはkernによるツメ組化対応のOS標準書体はNoto/源ノ・游・ヒラギノしかないので見慣れない人が多いのも仕方ないですが、慣れてくると**ベタ組の見出しの仮名の文字間余白が気になる**ようになります。Word・パワポもさっさと対応してもらいたいですしCSSでもfont-feature-settings以外で指定できるようになってもらいたいですね。

~~追記：font-features-settings: "palt"ではなくfont-kerning: normalで詰めてる？font-kerning: normalは以下の通り日本語仮名に対する動作を想定していないような気がします~~

https://github.com/w3c/csswg-drafts/issues/6723

**Reply by tsubolabo** [reply]
混戦模様になってまいりました。tsubolabo は👍印が 0 で苦戦中のようです。
（誤解釈があればご指摘ください）

|            | タイトル | 見出し   |本文      | 👍印     |
|:-----------|:--------:|:--------:|:--------:|:--------:|
| tsubolabo  | ベタ     | ベタ     | ベタ     |    0     |
| Phroneris  | ？？     | ツメ     | ツメ     |    1     |
| tats-u     | ツメ     | ツメ     | ベタ     |    1     |

理由：
tsubolabo
　従来の様式でベストになるように工夫しているから。
Phroneris
　？？
tats-u
　「長文ではベタ組の方が読みやすい」が定説だから、本文はベタ。

**Reply by Phroneris** [reply]
う～ん、現実的にはいかに日本語主体の媒体といえどこの現代では英数字 0a/@ や絵文字 👍🎉😄 がバンバン混じり、ましてやプログラミング系CGMたるQiitaでは `` `インラインコード` `` や `$インライン数式$` まで公式機能としてバリバリ使われる以上、長文とて原稿用紙のような列の揃いをどこまで期待すべきかと考えると私はかなり疑問です。
そして実際、現状のQiitaも見出し・本文を問わずフォント指定の先頭にYaku Han JPを入れているくらいなのだから、公式的には詰める側の考えなのではないかと勝手に推察します。
（ってよく考えたらYaku Han JPが適用されてれば約物の不統一は起きてないはずだな…これWebフォントじゃなかったんだ…）

もちろん、たとえ横書きWebページでも、フォーマットを掌握または強制できる新聞や雑誌みたいなメディアなら等幅で整列している方が見やすいに決まってますけどね。
とはいえ、報道系Web記事でURL表記にすら `ｈｔｔｐｓ：／／` っつって全角英数字を使わされているのがあるとさすがにあーあって思いますが…いやこの話関係無いな。
関係ある話をすると、新聞や雑誌の見出しってスポーツ紙や週刊誌なら着色して縁取りして文字同士重ねてと何でもありですし、そこまで自由じゃない普通の白黒の新聞ですら縦横比の変更ぐらいは平気でやりますから、仮にもプレーンテキストの延長線上にあるMarkdownの見出しとは並列視が難しい気がします。
紙面の見出しは半歩ぐらいはロゴデザインの領域にあるというか、あるいは本文等幅のCGMだとしても「投稿」「キャンセル」ボタンにはプロポーショナルを使い得るみたいな、「見出し＝短文＝ツメ組」という図式は字面通りの `<h1>`～`<h6>` を指すというよりは限りある領域の中に情報を収めてデザイナーが成果物を（「法則を」ではなく）監修する世界の話のように見えます。

まぁ欧文とかで揃わなくなる旨は挙げられているモリサワのnote記事にも当然注釈されていますし、皆様の見識を拝見していると主義主張以前に私の専門的詳しさは明らかにこの場の平均より下ぐらいなので、釈迦に説法だとは思い恐縮なのですが…。
元々メイリオでやっていたサイトですし、私としては日本語文字をベタで行くならそれでも構わず、どっちかならプロポーショナルの方が好きぐらいの立場です。
ただし不統一だけはちょっと好きになれそうにありません…。タイトルと見出しの不統一はもちろんですが、見出しと本文のそれもちょっと…。

メイリオ時代にMac等でメイリオ以外で見ていた方々がこの不統一を既に浴びていたのかは気になります。
元から見出しだけ詰めていたんですかね？

**Reply by ohakutsu** [reply]
@tsubolabo @Phroneris @tats-u 
みなさん、たくさんのご意見ありがとうございます。
ご意見いただいている箇所については、フォント変更の影響が出ていると思います。

現在、タイトル、見出しや本文などのツメ組みとベタ組みついて、Qiitaとしてどのような形が望ましいかを模索している段階です。
直近の計画としては、特定の箇所だけを個別に変更するのではなく、まずは全体のバランスを踏まえたフォント調整を行いたいと考えています🙇

そのため、すべてのご要望にお応えできない場合もありますが、いただいたご意見は今後の検討に活かしていきますので、ご理解いただけると幸いです。
